src/Views/View.vala

```diff
@@ -18,6 +18,8 @@
  * Authored by: Corentin Noël <corentin@elementaryos.org>
  */
 
+public delegate void Fn ();
```

Comment:
`Fn` seems a little bit too ambigious. Perhaps `BeforeDestroyFn`?

Reply:
This is because it has no meaning besides serving as a generic type for any function / closure which takes no inputs and provides no outputs. It therefore may be used anywhere where that pattern makes sense. Was looking to emulate the `Fn()` trait in Rust.